### PR TITLE
Return appropriate gamepad id depending on headset vendor

### PR DESCRIPTION
**Pull request by dmarcos**
I made Rift and Quest ids the same since there's not available model at the moment for libraries to integrate accordingly and button mapping is identical. We will have to differentiate in the future.

### CODE REVIEW

File: src/VR.js [hunk 1]
```diff
@@ -610,37 +610,54 @@ class FakeVRDisplay extends VRDisplay {
...
+  }
+  return controllerIDs[vrDisplay.id + hand.charAt(0).toUpperCase() + hand.slice(1)];
+}
+
 let gamepads = null;
 function getGamepads(window) {
   const {oculusVRDisplay, openVRDisplay, oculusMobileVrDisplay, magicLeapARDisplay} = window[symbols.mrDisplaysSymbol];
-  if (
+  const presentingDisplay =
     GlobalContext.fakeVrDisplayEnabled ||
```
Comment (modulesio) [inline src/VR.js]: It wouldn't be the `presentingDisplay` in the fake vr display case based on this line. We should probably point to the actual fake VR display in the case it's detected.

Comment (dmarcos) [reply src/VR.js]: Done. Not sure if correct. not sure how to test that code path.

File: src/VR.js [hunk 2]
```diff
@@ -610,37 +610,54 @@ class FakeVRDisplay extends VRDisplay {
...
+  OculusQuestLeft: 'Oculus Touch (Left)',
+  OculusQuestRight: 'Oculus Touch (Right)',
+  OculusGo: 'Oculus Go',
+  GearVR: 'Gear VR',
+  OpenVR: 'OpenVR Gamepad',
+  OpenVRTrackerID: 'Tracker'
 };
 
+function getControllerID(vrDisplay, hand) {
+  if (!vrDisplay) { return; }
```
Comment (modulesio) [inline src/VR.js]: We always have a `vrDisplay` when requesting a controller ID so there's probably no reason to check this, right?

File: src/VR.js (hunk 1 above)
Comment (modulesio) [reply src/VR.js]: For posterity, it would be running `examples/realitytabs.html`, in WebXR mode.

File: src/VR.js (hunk 2 above)
Comment (dmarcos) [reply src/VR.js]: done

**Comment by modulesio** [comment]
Thanks!

**Pull request merged**